app-service-template/res/configuration.toml

```diff
@@ -105,11 +112,20 @@ TokenFile = "/tmp/edgex/secrets/new-app-service/secrets-token.json"
   Port = 59881
 
   [Clients.core-command]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 59882
-
-  [Clients.support-notifications]
+#  Protocol = "http"
+#  Host = "localhost"
+#  Port = 59882
+  # Choose to use new messaging based communication for Command Client.
+  # Only valid if MessageBus Trigger is used
+  # If prefer REST, remove below and uncomment above
+  UseMessageBus = true
+  [Clients.core-command.Topics]
+  QueryRequestTopic = "edgex/core/commandquery/request"
+  QueryResponseTopic = "edgex/core/commandquery/response/#"
+  CommandRequestTopicPrefix = "edgex/core/command/request"
+  CommandResponseTopic = "edgex/core/command/response/#"
```

Comment:
@hahattan , please confirm these values. I am getting timeout on the Query before able to do the Get Command. THX!

Reply:
Humm, they seem correct getting these debug log message in Command service, but always messaging Command Client always times out waiting on the response. 
```
level=DEBUG ts=2022-10-20T00:06:24.6432707Z app=core-command source=internal.go:195 msg="Command query request received on internal MessageBus. Topic: edgex/core/commandquery/request/sample-numeric, Correlation-id: f3279f21-d05d-4cb2-8f25-4dd58440659e "
level=DEBUG ts=2022-10-20T00:06:24.6468332Z app=core-command source=internal.go:217 msg="Command query response sent to internal MessageBus. Topic: edgex/core/commandquery/response, Correlation-id: f3279f21-d05d-4cb2-8f25-4dd58440659e"
```